tests/installation/start_install.pm

```diff
@@ -134,4 +134,18 @@ sub run {
     }
 }
 
+sub post_fail_hook {
+    my $self = shift;
+    if (check_var('UPGRADE', '1')) {
+        select_console 'log-console';
+        save_screenshot;
```

Comment:
Any reason save_screenshot here?

Reply:
In some cases, the curl upload will fail, saving pictures here will help us to find the reason
quickly.